src/stepper/index.tsx

```diff
@@ -43,6 +51,7 @@ export default class Stepper extends PureComponent <StepperProps, StepperState>
 
   componentDidMount() {
     this.update(this.state.value, undefined, false)
+    document.addEventListener('keydown', this.handleKeyDown)
```

Comment:
Can you use a native react event for this?

Reply:
There are no other way to manage those events, I plan to remove it cause in case of multiple Stepper it will be a mess.

Reply:
Done